styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blueviolet;
+}
```

Comment:
- [ ] Here you can target the class name instead of the actual tag. because it is a good way of the code flow.